### PR TITLE
feat(protocol-designer): remove redesign feature flag

### PR DESCRIPTION
**Pull request by koji**
# Overview
remove redesign feature flag

close AUTH-940

## Changelog
- remove redesign ff
- remove codes that are related to moam 

## Risk assessment
low

### CODE REVIEW

File: protocol-designer/src/feature-flags/types.ts [hunk 1]
```diff
@@ -30,7 +30,6 @@ export type FlagTypes =
   | 'OT_PD_DISABLE_MODULE_RESTRICTIONS'
   | 'OT_PD_ALLOW_ALL_TIPRACKS'
   | 'OT_PD_ENABLE_ABSORBANCE_READER'
-  | 'OT_PD_ENABLE_REDESIGN'
```
Comment (jerader) [inline protocol-designer/src/feature-flags/types.ts:33]: nit: can we add this to the list of deprecated flags on line 5?

File: protocol-designer/src/ProtocolEditor.tsx [hunk 2]
```diff
@@ -1,81 +1,29 @@
...
-import { LabwareUploadMessageModal } from './components/modals/LabwareUploadMessageModal/LabwareUploadMessageModal'
-import { GateModal } from './organisms/GateModal'
-import { CreateFileWizard } from './components/modals/CreateFileWizard'
-import { AnnouncementModal } from './organisms'
 import { ProtocolRoutes } from './ProtocolRoutes'
 import { useScreenSizeCheck } from './resources/useScreenSizeCheck'
 import { DisabledScreen } from './organisms/DisabledScreen'
 
-import styles from './components/ProtocolEditor.module.css'
 import './css/reset.module.css'
```
Comment (jerader) [inline protocol-designer/src/ProtocolEditor.tsx]: do we still need this?

Comment (koji) [reply protocol-designer/src/ProtocolEditor.tsx]: i'll remove that

File: protocol-designer/src/feature-flags/types.ts [hunk 3]
```diff
@@ -23,14 +23,15 @@ export const DEPRECATED_FLAGS = [
   'OT_PD_ALLOW_96_CHANNEL',
   'OT_PD_ENABLE_FLEX_DECK_MODIFICATION',
   'OT_PD_ENABLE_MULTI_TIP',
+  'OT_PD_ENABLE_MOAM',
```
Comment (jerader) [inline protocol-designer/src/feature-flags/types.ts:26]: thanks for adding this as well!

**Review by jerader — APPROVED** [review]
lgtm! exciting to see this ff gone 🗑️

**Pull request merged**